src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+/**
+ * Represents a command class that adds a task.
+ * 
+ * @author Sherman Ng Wei Sheng
+ */
+public class AddCommand extends Command {
+    
+    private boolean isExit;
+    private Task task;
+
+    /**
+     * Constructor for AddCommand.
+     * 
+     * @param task Task to be added.
+     */
+    public AddCommand(Task task) {
+        this.task = task;
+        this.isExit = false;
+    }
+
+    /**
+     * Returns true if the command is a programme terminating command.
+     * 
+     * @return True if it is a terminating command and false otherwise.
+     */
+    @Override
+    public boolean isExit() {
```

Comment:
Maybe getIsExit() is a better choice for this method name, as isExit sounds like a boolean name to me.

Reply:
EDIT: My bad seems like this is the correct naming convention for boolean methods.